sources/APUTDQ

```diff
@@ -132,12 +134,14 @@ Copyright (c) 1981-1988, 1990, 2021-2022 by Venue & Xerox Corporation.
 (DEFINEQ
 
 (LOADUP
-  [LAMBDA (FILES)                                      (* ; "Edited 12-Mar-2021 00:15 by larry")
-    (for X in FILES do (PRINTOUT T "loading " X T)
-                                  (OR (FMEMB X SYSFILES)
-                                      (DOFILESLOAD (LIST '(SYSLOAD)
-                                                         X)))
-                                  (SMASHFILECOMS X])
+  [LAMBDA (FILES)                                            (* ; "Edited 16-Jan-2025 13:35 by mth")
+                                                           (* ; "Edited 12-Mar-2021 00:15 by larry")
+    (for X in FILES do (if (FMEMB X SYSFILES)
+                           then (PRINTOUT T X " already loaded" T)
```

Comment:
OK, and we should also fix the loadup files so that it does not generate this note -  we should load UNIXUTILS before we load things that depend on it.